src/Exception/RouteNotFound.php

```diff
@@ -2,8 +2,6 @@
 
 namespace Penny\Exception;
 
-use Exception;
-
 final class RouteNotFound extends Exception
```

Comment:
Is not better with `RouteNotFoundException` in order to be specific with `Exception` ?